src/microbe_stage/MicrobeStage.tscn

```diff
@@ -1945,8 +1849,6 @@ anims/ShowLeftPanels = SubResource( 41 )
 [connection signal="pressed" from="MicrobeHUD/BottomBar/QuestionMarkButton" to="MicrobeHUD" method="ToggleHelpScreen"]
```

Comment:
Since the `ToggleHelpScreen` method on MicrobeHUD was removed, the question mark button couldn't toggle the help screen.

Reply:
It should now work again.